src/services/offer.js

```diff
@@ -6,6 +6,10 @@ class OfferService {
 
     }
 
+    static get getLimit() {
```

Comment:
Maybe a more descriptive name? `MAX_OFFERS_PER_QUERY` - All caps because it is constant and static

currently getLimit could refer to the limit of offers posted by a company, some sort of end date,... this way it's more specific. Also the **get**Limit makes it seem like it is some variable field in the OfferService. Being a static constant, I suggest the above.

Reply:
Fixed